applications/nrf_desktop/doc/info.rst

```diff
@@ -0,0 +1,42 @@
+.. _info:
+
+Info module
+###########
+
+Use the Info module to:
+
+* handle the role of the final subscriber for the :ref:`config_channel` events,
+* provide the board name through the :ref:`config_channel`.
+
+Module Events
+*************
+
++------------------+--------------------------------+---------------+------------------------+------------------+
+| Source Module    | Input Event                    | This Module   | Output Event           | Sink Module      |
++==================+================================+===============+========================+==================+
+| :ref:`hids`      | ``config_event``               |  ``info``     |                        |                  |
++------------------+                                |               |                        |                  |
+| :ref:`usb_state` |                                |               |                        |                  |
++------------------+--------------------------------+               |                        |                  |
+| :ref:`hids`      | ``config_fetch_request_event`` |               |                        |                  |
++------------------+                                |               |                        |                  |
+| :ref:`usb_state` |                                |               |                        |                  |
++------------------+--------------------------------+               +------------------------+------------------+
+|                  |                                |               | ``config_fetch_event`` | :ref:`hids`      |
+|                  |                                |               |                        +------------------+
+|                  |                                |               |                        | :ref:`usb_state` |
++------------------+--------------------------------+---------------+------------------------+------------------+
+
+Configuration
+*************
+
+The Info module is required by the :ref:`config_channel`.
+The module is enabled with the same Kconfig option as the :ref:`config_channel` (``CONFIG_DESKTOP_CONFIG_CHANNEL_ENABLE``).
+
+Implementation details
+**********************
+
+The Info module must be the final subscriber for the ``config_event`` and ``config_fetch_request_event``.
+This is required by the :ref:`config_channel` implementation.
```

Comment:
Why is that? What if it is not set as the final subscriber?

Reply:
The modules will not be indexed properly, so they may not be accessible through the config channel. I will add this information.